nextjs-13/src/components/ListNewsletter/index.tsx

```diff
@@ -1,24 +1,27 @@
+import NewsletterItem from '@components/NewsletterItem';
+
 interface Record {
   id: string;
   name: string;
 }
 
 interface Props {
   records: Record[];
+  onAfterCloseCallback: () => undefined;
 }
 
-const ListNewsletter = ({ records }: Props) => {
+const ListNewsletter = ({ records, onAfterCloseCallback }: Props) => {
```

Comment:
I understand the need to pass a callback to refresh the list of newsletter (if we were to use Redux or useReducer we would not need to pass down so many props) but this naming is quite confusing as it related to the modal (which is a detail of implementation). So I recommend the following:

```suggestion
const ListNewsletter = ({ records, refreshRecordListCallback }: Props) => {
```

Below in https://github.com/nimblehq/js-framework-benchmark/pull/116/files#diff-223e9b714cf2c9b5f163079f0bd63d3a6eac9c46915fc3afde20a2742897d706R70, the naming makes sense as it related to the modal but not in `ListNewsletter`.

Reply:
Yes that's my bad 😭 Thanks for reminding me!

Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/88c0be267f1152993e680d082a339f3909afd2d2)